.circleci/config.yml

```diff
@@ -22,29 +22,60 @@ jobs:
           covermode: atomic
           failfast: true
           race: true
+  build_extension:
```

Comment:
Looks like we're adding the `build_extension` command here, wasn't this in #6?

Reply:
Nope, the `build_extension` step was what was missing from #6. #6 introduced publish_s3 but errored out because ~/artifacts didn't exist.